cmd/lotus-storage-miner/info.go

```diff
@@ -355,3 +370,40 @@ func sectorsInfo(ctx context.Context, napi api.StorageMiner) error {
 
 	return nil
 }
+
+func producedBlocks(ctx context.Context, count int, head *types.TipSet, maddr address.Address, napi api.FullNode) error {
```

Comment:
How long does it take for 1 day / week / month?

Reply:
it depend on how many produced block one day. 
when miner produce 10-20 blocks one day, it take less 3 minutes for one day include stat produced blocks and all gas cost(need message replay, maybe cost most time).